tests/h/cli/commands/search_test.py

```diff
@@ -44,8 +45,7 @@ def test_handles_runtimeerror(self, cli, cliconfig, update_index_settings):
 
     @pytest.fixture
     def update_index_settings(self, patch):
-        config = patch('h.cli.commands.search.config')
```

Comment:
This was patching the whole module instead of just the one function so it was causing the ANNOTATION_MAPPING to get mocked hence the change to just mock the function under test.